src/core/audio/AudioSdl.cpp

```diff
@@ -77,11 +82,19 @@ AudioSdl::AudioSdl( bool & _success_ful, AudioEngine*  _audioEngine ) :
   	SDL_AudioSpec actual; 
 
 #ifdef LMMS_HAVE_SDL2
-	m_outputDevice = SDL_OpenAudioDevice (nullptr,
-										  0,
-										  &m_audioHandle,
-										  &actual,
-										  0);
+	const auto playbackDevice = ConfigManager::inst()->value(SectionSDL, PlaybackDeviceSDL);
+	const bool isDefaultPlayback = playbackDevice.isEmpty();
+
+	// Try with the configured device
+	const auto playbackDeviceCStr = isDefaultPlayback ? nullptr : playbackDevice.toLocal8Bit().data();
```

Comment:
```suggestion
	const auto playbackDevice = ConfigManager::inst()->value(SectionSDL, PlaybackDeviceSDL).toStdString();
	const bool isDefaultPlayback = playbackDevice.empty();

	// Try with the configured device
	const auto playbackDeviceCStr = isDefaultPlayback ? nullptr : playbackDevice.c_str();
```
- Fix UTF-8 usage - SDL expects UTF-8, not a locale-dependent 8-bit encoding
- Fix dangling pointer to temporary `QByteArray` data

Reply:
Done with commit c1e5de5b3b9.